client/state/selectors/is-automated-transfer-failed.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import {
+	flowRight as compose,
+} from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { transferStates } from 'state/automated-transfer/constants';
+import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+
+/**
+ * Maps automated transfer status value to indication if transfer is failed
+ *
+ * @param {String} status name of current state in automated transfer
+ * @returns {?boolean} is transfer currently failed? null if unknown
+ */
+export const isFailed = status =>
+	status
+		? status === transferStates.CONFLICTS || status === transferStates.FAILURE
+		: null;
```

Comment:
That part is pretty much copied straight of `is-automated-transfer-active.js`.
I was not 100% sure of when the status could be falsy, but I thought of keeping the very same logic in place to avoid possible unforeseen bugs arising from this change.

So: `isAutomatedTransferActive` takes care of answering as:
- AT is active
- AT is not active
- AT has not been attempted

If this is the case, then yes: `hasAutomatedTransferFailed` will only need to answer yes/no.